shared/gh/partials/tenant-admin/admin-manage-orgunits-add-modal.html

```diff
@@ -0,0 +1,49 @@
+<div class="modal fade" id="gh-manage-orgunits-add-modal" tabindex="-1" role="dialog" aria-labelledby="modal-title" aria-hidden="true" data-id="<%- data.data.orgUnitId %>" data-groupid="<%- data.data.groupId %>">
+    <div class="modal-dialog">
+        <div class="modal-content">
+            <div class="modal-header">
+                <button type="button" class="close" data-dismiss="modal">
+                    <span aria-hidden="true">×</span>
+                    <span class="sr-only">Close</span>
+                </button>
+                <%
+                    var title = 'course';
+                    if (data.data.orgUnitType) {
```

Comment:
Is there a way to make this just `data.orgUnitType` ? Don't bother if it's too much work